holidays/countries/mauritius.py

```
@@ -0,0 +1,203 @@
+
+        # Ugadi.
+        self._add_gudi_padwa(tr("Ougadi"))
+
+        # Ganesh Chaturthi.
+        self._add_ganesh_chaturthi(tr("Ganesh Chaturthi"))
+
+        # Diwali.
+        self._add_diwali_india(tr("Divali"))
+
+
+class MU(Mauritius):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)